Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "rails", "~> 6.1.0"
+gem "rails"
```

Comment:
please don't remove these version requirements. all of them are there because we need to carefully test our code before updating them.
you are welcome to update rubocop and related things but we can't make bulk changes to dependencies like this.

Reply:
sure, I added the versions back 7d2fd5c5e40c50ac6776bdebae59bc71de5b03d9

I still think we should trust the tests and upgrade dependencies